src/v2/Apps/Conversation/Components/Details.tsx

```diff
@@ -129,21 +130,25 @@ export const Details: FC<DetailsProps> = ({
       flexDirection="column"
       justifyContent="flex-start"
       height={[
-        "calc(100% - 115px)",
-        "calc(100% - 115px)",
-        null,
-        "calc(100% - 145px)",
+        "calc(100% - 122px)",
+        "calc(100% - 175px)",
+        "calc(100% - 122px)",
+        "calc(100% - 122px)",
         "100%",
       ]}
       flexShrink={0}
-      position={["absolute", "absolute", "absolute", "absolute", "static"]}
+      position={["fixed", "fixed", "fixed", "fixed", "static"]}
```

Comment:
`absolute` -> `fixed` fixed this with 'translated' detail box creating extra white space on the right side when detail box is closed:


<details><summary>Screenshots</summary>

White space and scrollbar on the bottom:

![Screen Shot 2020-08-20 at 2 24 02 PM](https://user-images.githubusercontent.com/687513/90810777-81bf1200-e2f1-11ea-9b1c-daac229666a0.png)

The 'hidden' detail box is there:

![Screen Shot 2020-08-20 at 2 27 13 PM](https://user-images.githubusercontent.com/687513/90810788-88e62000-e2f1-11ea-8fce-28e76f2f1c1e.png)

After: Scrollbar is gone!

![Screen Shot 2020-08-20 at 2 30 35 PM](https://user-images.githubusercontent.com/687513/90810971-d19dd900-e2f1-11ea-9c90-13fedcb5a442.png)

</details>